contrib/vagrant/scripts/helpers.bash

```diff
@@ -67,8 +67,8 @@ cluster_dns_ipv6=${K8S_CLUSTER_DNS_IPV6:-"FD03::A"}
 cluster_api_server_ipv4=${K8S_CLUSTER_API_SERVER_IPV4:-"172.20.0.1"}
 cluster_api_server_ipv6=${K8S_CLUSTER_API_SERVER_IPV6:-"FD03::1"}
 
-k8s_version="v1.26.0-rc.0"
-etcd_version="v3.5.5"
+k8s_version="v1.27.0"
+etcd_version="v3.5.7"
```

Comment:
I remember we had issues with this etcd version - https://github.com/cilium/cilium/issues/23760 something about the image being distroless etc- was that fixed in the meantime?

Reply:
This is just for testing. It doesn't actually use the docker image either. This version slug is used to download etcd and runs it as a systemd process.